db/queries/assessment_records/queries.py

```diff
@@ -57,11 +61,31 @@ def get_metadata_for_fund_round_id(
     if asset_type != "ALL" and asset_type != "":
         statement = statement.where(AssessmentRecord.asset_type == asset_type)
 
-    if status != "ALL" and status != "":
+    if status not in ("", "ALL", "FLAGGED"):
```

Comment:
Could we do a pattern match on status rather than a series of if statements i.e. https://peps.python.org/pep-0634/?